packages/roosterjs-editor-plugins/lib/plugins/ImageSelection/ImageSelection.ts

```diff
@@ -47,7 +46,7 @@ export default class ImageSelection implements EditorPlugin {
     }
 
     onPluginEvent(event: PluginEvent) {
-        if (this.editor && this.editor.isFeatureEnabled(ExperimentalFeatures.ImageSelection)) {
+        if (this.editor) {
```

Comment:
please also mark this feature as deprecated from its definition, follow the pattern of other graduated features. Also remove it from demo site